app/assets/stylesheets/application.scss

```diff
@@ -6,5 +6,5 @@ $govuk-use-legacy-palette: false;
 @import 'components/*';
 @import 'finder_frontend';
 @import 'views/search';
-@import 'views/qa';
+@import 'views/brexit_checker_question_page';
```

Comment:
This is a rename - I removed all the qa related css, but there was one class that still seemed relevant to the brexit_checker, so i've renamed it and stuck it in it's own file below.